src/PortingAssistant.Client/Program.cs

```diff
@@ -40,9 +40,9 @@ static void Main(string[] args)
                 {
                     var configuration = new PortingAssistantConfiguration();
                     var roamingFolder = Environment.GetFolderPath(Environment.SpecialFolder.ApplicationData);
-                    var driftLogs = Path.Combine(roamingFolder, "Porting Assistant for .NET", "logs", "drift");
-                    var logFilePath = Path.Combine(driftLogs, "portingAssistant-assessment.log");
-                    var metricsFilePath = Path.Combine(driftLogs, "portingAssistant-assessment.metrics");
+                    var driftLogs = Path.Combine(roamingFolder, "Porting Assistant for .NET", "logs");
+                    var logFilePath = Path.Combine(driftLogs, "portingAssistant-drift.log");

```

Comment:
This is open sourced CLI, we should not name it drift just because it's currently used by Drift. Please avoid using `drift` for any of the variable names, open source community will not know what `drift` means, this will only creates confusion. Suggest to use something like `portingAssistant-client-cli.log`.

Reply:
update it to general client name